packages/wallets/src/AppContent.tsx

```diff
@@ -26,7 +28,7 @@ const AppContent: React.FC = () => {
     }, []);
 
     return (
-        <div className='wallets-app'>
+        <div className='wallets-app' key={i18n.language}>
```

Comment:
```suggestion
        <div className='wallets-app' key={`wallets_app_${i18n.language}`}>
```
better to be more unique